deps/libgit2.mk

```diff
@@ -104,8 +104,9 @@ $(LIBGIT2_SRC_PATH)/libgit2-remote-push-NULL.patch-applied: $(LIBGIT2_SRC_PATH)/
 	echo 1 > $@
 
 $(build_datarootdir)/julia/cert.pem: $(CERTFILE)
+	$(RM) $@
 	mkdir -p $(build_datarootdir)/julia
-	-cp $(CERTFILE) $@
+	cp $(CERTFILE) $@
```

Comment:
The man page for cp on linux states:

```
-f, --force
              if an existing destination file cannot be opened, remove it and try again (this option is ignored when the -n option is also used)
```

The man page from FreeBSD states:
```
     -f	   For each existing destination pathname, remove it and create	a new
	   file, without prompting for confirmation regardless of its permis-
	   sions.  (The	-f option overrides any	previous -i or -n options.)
```

`cp -f` works on my system as well, but I am not sure that the behavior is homogeneous.